source/get-started/virtual-machine-install/proxmox.rst

```diff
@@ -0,0 +1,211 @@
+.. _proxmox:
+
+|CL-ATTR| on Proxmox\* Virtual Environment 
+##########################################
+
+This guide explains how to create a new VM in Proxmox VE 6.1-3, install and run |CL| on as a guest OS.
+
+.. contents::
+   :local:
+   :depth: 1
+
+Prerequisites
+*************
+
+* Proxmox VE 6.1-3 server already set up and you have familiarity with how 
+  to use it.
+
+Download the Latest |CL| Live Server Image
+******************************************
+
+#. Visit our `Downloads`_ page.
+
+#. Download the file :file:`clear-<release number>-live-server.iso`,
+   also called the |CL| Server.
+
+   .. note::
+
+      <release-number> is the latest |CL| auto-numbered release.
+
+Upload |CL| Live Server Image to Promox Server
+**********************************************
+
+#. Connect to your Proxmox server and log into an account with sufficient
+   permission to create and manage VMs.
+
+#. Under the :guilabel:`Server View` window, select the :guilabel:`local` 
+   storage.  See Figure 1.
+
+#. On the right window, click :guilabel:`Upload`.
+
+   .. figure:: ../../_figures/proxmox/proxmox-01.png
+      :scale: 100%
+      :alt: Proxmox - Upload ISO
+
+      Figure 1: Proxmox - Upload ISO
+
+#. Set the :guilabel:`Content` as `ISO image". See Figure 2.
```

Comment:
Replace double quotation to tick mark ` ` ` after ISO image. 
One cause of failed build.